doc/reference/networking/net_context.rst

```diff
@@ -6,6 +6,8 @@ Networking Context
 Overview
 ********
 
+This API is not meant for application use. Application should use
```

Comment:
```suggestion
This API is deprecated. Applications should use
```

Reply:
Actually we are not deprecating it as it will be used internally in the stack. Here I am understanding that the "deprecating" word means that it is to be removed from the system, correct if I am wrong here. We are trying to push people to use the BSD socket API instead of net_context one.

Reply:
Ah, in that case, maybe we should delete the API documentation below (lines 12-EOF)?

Reply:
Sure, that makes sense.